cookbooks/crystal_preferred_orientation_olivine_fraters_billen_2021/doc/crystal_preferred_orientation_olivine_fraters_billen_2021.md

```diff
@@ -0,0 +1,119 @@
+# Olivine Fabric Developments Under Simple Shear
+
+*This section was contributed by Xiaochuan Tian, with helps from Yijun Wang and Menno Fraters. It is
+based on a section in {cite:t}`fraters_billen_2021_cpo`
+by Menno Fraters and Magali Billen published in 2021.*
+
+This cookbook explains how to set up a numerical experiment for fabric
+developments of a single olivine particle under simple shear macroscopic strain.
+It uses ASPECT's
+crystal preferred orientation (CPO) implementation, which is described in detail in
+{cite:t}`fraters_billen_2021_cpo`.
+The fabric calculation is based on DRex {cite:t}`kaminski2004`.
+This cookbook describe how to use ASPECT CPO implementation to calculate the major
+olivine fabrics observed in the lab experiments. It focuses on reproducing Fig. 3
+of {cite}`fraters_billen_2021_cpo`.
+
+## Motivation
+The Earth's plates can move relative to each other because the underlying mantle
+can be deformed to accommodate such motions.
+This motion reflects dynamics inside the planet but cannot be observed directly, as the depth of interest is beyond human drilling capability. However, indirect observations from seismic anisotropy provide information about the directionality of seismic wave speeds propagating inside the Earth. If such directionality is controlled by the mantle’s motion or deformation, we can use seismic anisotropy observations to infer mantle deformation.
```

Comment:
Currently it sounds like you say the plate motion cannot be observed directly, which is not correct. You probably mean the dynamics that cannot be observed. Maybe change to:
```suggestion
This motion reflects dynamics inside the planet, which cannot be observed directly, as the depth of interest is beyond human drilling capability. However, indirect observations from seismic anisotropy provide information about the directionality of seismic wave speeds propagating inside the Earth. If such directionality is controlled by the mantle’s motion or deformation, we can use seismic anisotropy observations to infer mantle deformation.
```